project_argparser.py

```diff
@@ -100,21 +101,26 @@ def open_car_dict(self, input):
         
         return car_dict
     
-    #opens csv file and returns a dict with keys "air_density" and integers representing breakpoints
+    # Opens a csv file and returns a matrix with rows (first index) corresponding to the headers 
+    # of the TUM track csv format. Left to right corresponds to 0 to 8.
     def open_track_dict(self, input):
-        if input == None:
-            input = self.off_msg
+
+        input = "tracks/" + input
```

Comment:
I think these file paths are better called out explicitly instead of adding on the `track` directory automatically. The reason for that is so we can call out other track files that aren't necessarily in our `track` directory.